lib/doorkeeper/oauth/refresh_token_request.rb

```diff
@@ -10,9 +10,9 @@ class RefreshTokenRequest < BaseRequest
       validate :client,       error: :invalid_client
       validate :client_match, error: :invalid_grant
       validate :scope,        error: :invalid_scope
+      validate :resource_indicators, error: :invalid_target
```

Comment:
this is really small, but if all `error`s are in one line, maybe it's worth doing the same with a new one? or remove this completely :)